problems/countNumbers.js

```diff
@@ -12,8 +12,11 @@
  * returns { 99: 2, 11: 1, 12: 1, 13: 1, 58: 1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+    arr = {[]}
+    for(let i = 0; i < arr.length; i++){
+        return
```

Comment:
What is this function asking us to return? Do we want to return inside of our for loop?